script.js

```diff
@@ -1,11 +1,18 @@
+var colors = ["red", "blue", "yellow"];
 // Create an array called `colors` that contains three colors.
-
+console.log(colors[2]);
 // Log the last item in the array to the console.
-
+//for (i = 0; i < colors.length; i++) {
+  //  text+= "My favorite color is" + colors[i];
+  //};
```

Comment:
you're close `console.log("My favorite color is" + colors[i]);`. `x += y` is the equivalent of `x = x + y`